dspace-server-webapp/src/main/java/org/dspace/app/rest/authorization/AuthorizationRestUtil.java

```diff
@@ -66,7 +67,8 @@ public BaseObjectRest getObject(Context context, String id) throws SQLException
         String[] objType;
         try {
             objType = parts[2].split("\\.");
-            DSpaceRestRepository repository = utils.getResourceRepositoryByCategoryAndModel(objType[0], objType[1]);
+            DSpaceRestRepository repository = utils
+                .getResourceRepositoryByCategoryAndModel(objType[0], English.plural(objType[1]));
```

Comment:
why is this required? this seems to be against the goal of this PR to remove the dependency from a computation of the plural

Reply:
@alexandrevryghem : Did [this question](https://github.com/DSpace/DSpace/pull/9241#discussion_r1479312626) get answered?

Reply:
@abollini: thnx for the feedback, I implemented the changes you requested. For the remaining `English.plural`, I did not change it because this would also require all the feature's `getSupportedTypes` methods to be updated as well. I fixed it on a separate branch if you want to include this in this PR too: https://github.com/alexandrevryghem/DSpace/commit/2cf979260aac48c9742d9e4f0d46ea0645855877. I can also open a new PR after this one is merged with these changes, this way we can already merge the process polling PRs 🤷‍♂️

Reply:
it is ok for me to open a separate PR for this additional refactoring (to be considered for DSpace 8 or 9)